app/js/views/reminders.jsx

```diff
@@ -74,29 +74,45 @@ export default class Reminders extends React.Component {
   onReminder(evt) {
     const reminder = evt.result;
 
-    const reminders = this.state.reminders;
-    reminders.push({
-      id: reminders.length,
-      recipients: reminder.users,
-      content: reminder.action,
-      datetime: reminder.time,
-    });
-
-    this.setState({ reminders });
-
+    // @todo Nice to have: optimistic update.
     this.server.reminders
       .set({
         recipients: reminder.users,
         action: reminder.action,
         due: Number(reminder.time),
       })
+      .then((reminder) => {
```

Comment:
Yep exactly. I just need to find a way to hold a reference to the newly created element and inject its ID later when we receive it from the server.